index.js

```diff
@@ -313,6 +315,17 @@ function Argv (processArgs, cwd) {
     return options
   }
 
+  var groups = {'Options:': []}
```

Comment:
I like your solution in https://github.com/nexdrew/yargs/commit/dc350f2bfb17de2aaa49c9313d57296295a235af, will cherry pick over.